tmt/steps/execute/internal.py

```diff
@@ -285,7 +302,7 @@ def go(self, guest):
 
         self._run_tests(guest)
 
-    def _run_tests(self, guest, extra_environment=None):
+    def _run_tests(self, guest: Guest, extra_environment: EnvironmentType = {}) -> None:
         """ Execute tests on provided guest """
         extra_environment = extra_environment or {}
```

Comment:
This line might not be needed since the signature of the method got changed. The `extra_environment` variable value will always be either passed in the method parameter or set to `{}`.

Reply:
thanks, I agree it's not needed anymore.